tests/test__collection_api.py

```diff
@@ -3068,6 +3068,17 @@ def test__aggregate_project_missing_fields(self):
         ])
         self.assertEqual([{}], list(actual))
 
+    def test__aggregate_project_missing_nested_fields(self):
```

Comment:
Please add an equivalent test in `test__mongomock` to ensure the behavior is kept the same between `pymongo` and `mongomock`

Reply:
done